executor/aggregate.go

```diff
@@ -652,28 +655,28 @@ func (e *HashAggExec) parallelExec(ctx context.Context, chk *chunk.Chunk) error
 	if e.executed {
 		return nil
 	}
-	for !chk.IsFull() {
-		e.finalInputCh <- chk
+	for {
 		result, ok := <-e.finalOutputCh
-		if !ok { // all finalWorkers exited
+		if !ok {
 			e.executed = true
-			if chk.NumRows() > 0 { // but there are some data left
```

Comment:
If there's remaining data in `chk`. The `e.isChildReturnEmpty` must be false. Thus we don't need to return it here.